packages/dds/merge-tree/src/test/snapshotlegacy.spec.ts

```diff
@@ -63,7 +63,8 @@ describe("snapshot", () => {
                 logger: client2.logger,
                 clientId: (i + 1).toString(),
             };
-            await client2.load(runtime as IFluidDataStoreRuntime, services, serializer);
+            const result = await client2.load(runtime as IFluidDataStoreRuntime, services, serializer);
+            await result.catchupOpsP;
```

Comment:
Why is this extra await here needed?

Reply:
because the load in line 66 was partial load, i think it only loaded the header. the await is for the body

Reply:
I guess then why do we have to wait for the body to load now? what changed for it to require the body to load?